.github/workflows/nightly-integration.yml

```diff
@@ -0,0 +1,68 @@
+# Run integration tests nightly on main
+
+# !! Relevant changes to this file should be propagated manual-integration.yml
+
+name: nightly-integration-main
+on:
+  schedule:
+    # run every day at 03:00 (AM)
```

Comment:
Is it 3AM in UTC?

Reply:
Correct, it is UTC. Added it to the comment